### PR TITLE
update localization spec and also close the open questions

### PR DESCRIPTION
**Pull request by OsmondJiang**
try to close the open questions and also update the spec according to the loc team's confirmation.

### CODE REVIEW

**Comment by OsmondJiang** [comment]
#3623

**Comment by OsmondJiang** [comment]
#3688

**Comment by mingwli** [comment]
image

this need to be changed?

File: docs/designs/conceptual-localization.md [hunk 1]
```diff
@@ -77,22 +77,22 @@ Below kinds of mappings are considered to be supported and there is a **strong c
 
     Here is an string convention for loc repo name:
```
Comment (yufeih) [inline docs/designs/conceptual-localization.md]: In the `Repository` convention, we only need `{source-repo-name} -> {source-repo-name}.{locale}`. Repos with `{source-repo-name}.{source-locale} -> {source-repo-name}.{loc-locale}` convention will be renamed.

File: docs/designs/conceptual-localization.md [hunk 2]
```diff
@@ -77,22 +77,22 @@ Below kinds of mappings are considered to be supported and there is a **strong c
...
     source repo       ->locale->      localization repo
-    dotnet/docfx        zh-cn         dotnet/docfx.localization
-    dotnet/docfx        de-de         dotnet/docfx.localization
+    dotnet/docfx        zh-cn         dotnet/docfx.loc
+    dotnet/docfx        de-de         dotnet/docfx.loc
     branch mapping example:
     source branch           -->           localization branch
-    master                  -->           zh-cn-master
-    live                    -->           de-de-live
+    master                  -->           master.zh-cn
```
[the remark this answers is not in the document]
Comment (yufeih) [reply docs/designs/conceptual-localization.md]: `master-sxs.zh-cn`

File: docs/designs/conceptual-localization.md (hunk 1 above)
Comment (yufeih) [reply docs/designs/conceptual-localization.md]: yes, mooncake repos will be renamed as well.

Comment (OsmondJiang) [reply docs/designs/conceptual-localization.md]: sure, will remove this convention

**Pull request merged**